packages/terra-table/tests/jest/SingleSelectableRows.test.jsx

```diff
@@ -29,12 +29,6 @@ it('should render SingleSelectableRows with one row', () => {
   expect(tableRows).toMatchSnapshot();
 });
 
-it('should render SingleSelectableRows with maximum height set', () => {
```

Comment:
Why did we remove this test?

Reply:
It was actually your recommendation. It is a stale test left by [this pull request removing max height tests](https://github.com/cerner/terra-core/pull/238)